### DISCUSSION TITLE
imagemagick reads an image with wrong dpi

### DISCUSSION THREAD

**Discussion by gitexplr** [dlemstra/Magick.NET#628] [General] [answered]
I have a png image that I need to modify using imagemagick, original image has 300dpi. But when used imagemagick to read the image, density shows as 762 which is causing wrong size of output.

**Comment by dlemstra** [comment]
Can you share your image? And which version of Magick.NET/ImageMagick are you using?

**Reply by gitexplr** [reply]
Thanks for the quick reply.

I am using version 7.0.5.501. unfortunately I can't share image.

**Reply by gitexplr** [reply]
I can try to email if you can provide your email.

**Reply by dlemstra** [reply]
My email can be seen on my GitHub profile. And the version of Magick.NET that you are using is pretty old. What happens when you upgrade to a more recent version?

**Reply by gitexplr** [reply]
I will try to use latest version and see.

**Comment by gitexplr** [comment]
Hello I tried using latest version, still saw the issue. Sent the image thru email, can you please check?

**Comment by dlemstra** [comment, marked as answer]
I am unable to reproduce your isue this is the output that I am getting with the test image that you provided:

```
image.Density
{118.11x118.11 cm}
    Units: PixelsPerCentimeter
    X: 118.11
    Y: 118.11
image.Density.ChangeUnits(DensityUnit.PixelsPerInch)
{299.9994x299.9994 inch}
    Units: PixelsPerInch
    X: 299.9994
    Y: 299.9994
```

Where do you see the value `762` as the DPI?

**Reply by gitexplr** [reply]
Sorry for the delay, original image is now modified. So didn't see the issue anymore. Thanks again for the help!!
